lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb

```diff
@@ -0,0 +1,21 @@
+class Reports::SubscriptionChangesAfterSwitchToDailyDigest
+  def self.call
+    updated_subscriptions = Subscription.where(source: 4).where.not(ended_at: nil)
+
+    newer_subscriptions = updated_subscriptions.all.to_a.flat_map do |s|
+      Subscription
```

Comment:
Could we do `Subscription.includes(:subscriber_list)` here and use the loaded relation to populate the subscriber list title on line 17? Slightly simplifies the code and means we can remove line 11.

Reply:
Thinking about it don't know how well the above suggestion would scale

Reply:
Yes we can/should 👍.

(I learnt about `index_by` recently, and it seems to have overridden my memory of `includes`!)